docs/COMPILER_VERSIONS.md

```diff
@@ -64,5 +64,5 @@ We currently use the following version:
 | Type                     | Rust version | Note                              |
 | ------------------------ | ------------ | --------------------------------- |
 | Production Rust compiler | 1.69.0       | Builders version 0016             |
-| Min Rust compiler        | 1.60.0       | Supports builder versions >= 0012 |
+| Min Rust compiler        | 1.65.0       | Supports builder versions >= 0012 |
```

Comment:
Question: What is the note about? Do we need to update that too?

Reply:
It is helpful, yeah. In https://github.com/CosmWasm/wasmvm/tree/main/builders#changelog you see the Rust versions of the builders. I.e. builders version >= 0014 can compile the codebase after the upgrade.

Reply:
Ah, thanks. That context was missing for me. Updated